bnelson777/id.ly
# IL-104: Be able to set a default identity from the view card screen of identities in wallet

When clicking on a card from the wallet, there will be a button on the view card screen allowing you to set that card as the default. When the card is set as default, it should be moved to the top of the wallet. If it is already at the top, a popup will inform you that it is already the default.

How to test:

- Go to your wallet and click on one of the cards besides the first one. Click on the "Set Default" button. You should be returned to your wallet and the card should now be on top.

- Click on your default card and click the "Set Default" button. A popup should appear and not allow you to set the card as default. 

- Add a new card. The new card should now be added at the end instead of the front.

- Go to the rolodex and click on a card. There should not be a button to set it as a default card.
